server/services/playoffs.py

```
@@ -20,22 +23,49 @@
+                season_nullable=season,
+                season_type_nullable="Playoffs",
+                league_id_nullable="00",
+            )
+        else:
+            games = LeagueGameLog(
+                season=season,
+                season_type_all_star="Playoffs",
+                league_id="00",
+            )
     except (ReadTimeout, RequestsConnectionError) as e:
         raise HTTPException(status_code=503, detail=f"NBA Stats API unavailable: {e}")
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)